umm-spec-lib/src/cmr/umm_spec/test/expected_conversion.clj

```diff
@@ -8,6 +8,7 @@
     [clojure.string :as str]
     [cmr.common.util :as util :refer [update-in-each]]
     [cmr.umm-spec.json-schema :as js]
+    [cmr.umm-spec.migration.version.core :as core]
```

Comment:
Hey, while you're in here, could you un-indent these requires by one space (to match our convention).